templates/web/fixmystreet.com/about/open311-api-info.html

```diff
@@ -0,0 +1,154 @@
+[% INCLUDE 'header.html', title = 'Open311', bodyclass = 'twothirdswidthpage' %]
+
+[% INCLUDE 'about/_sidebar.html' %]
+
+<h1>FixMyStreet Open311 API</h1>
+
+<h2>Route FixMyStreet reports directly into backend systems.</h2> 
+
+<p>By default, FixMyStreet reports are sent to authorities via email, which we understand is not always convenient for councils who are using backend systems to manage their fault reports.</p> 
+<p>Using the Open311 API, reports can be sent directly into backend systems, removing the need to receive emails, eliminating manual intervention and improving the user experience.</p>
+
+<dl id="client">
+
+<dt>What is Open311?</dt>
+
+<dd>
+<p>In a nutshell, it's a free, international technology, known as an open standard, that allows civic reporting services and systems to 'talk to' each other. </p>
+<p>Setting up an Open311 API endpoint for FixMyStreet will enable you to receive reports from FixMyStreet.com and the app directly into your backend system(s) instead of via email. It will also allow you to close/update reports on FixMyStreet.com or the app when work is completed.</p>
+<p>Open311 isn’t new (we’ve been using it to connect to council systems since 2011), but it still isn't as widely known as it deserves to be. You can find out lots more at <a href="http://open311.org/">open311.org</a>.</p>
+
+</dd>
+
+<dt>How do I get started with Open311?</dt>
+
+<dd>
+<p>Setting up an Open311 endpoint yourself to provide access to one or more of your current services shouldn’t take you more than a few days of development: it usually only needs the support of your technical team and some project management time.</p>
+<p>We’re happy to give you some advice if you are just getting started, and you can <a href="https://github.com/mysociety/fixmystreet/wiki/Open311-FMS---Complete-Spec">find our full Open311 specification here.</a></p>
+
+<ul>
+  <li>First, let us know you are planning to set up an Open311 endpoint by emailing support@fixmystreet.com. This way we know to expect your API key and can also provide some advice where appropriate.</li>
+  <li>Find everything you need to know in the complete tech spec for the Open311 endpoint here: https://github.com/mysociety/fixmystreet/wiki/Open311-FMS---Complete-Spec</li>
+  <li>Once you've got a functional Open311 testing endpoint setup, email us the URL and API key and we’ll configure our FixMyStreet staging site to send appropriate FixMyStreet reports to that endpoint. From here you can inspect the data you receive and make adjustments until you’re happy.</li>
+  <li>You can then implement the spec we have given you access to, this includes</li>
```

Comment:
The sublist here is part of this list item, so you don't want to close the list item here (so remove the `</li>`)

Reply:
Ok, I _think_ I've corrected this. I removed the </li> from after the word 'includes' - is that what you meant?